examples/reference/pandas/bar.ipynb

```diff
@@ -32,7 +30,7 @@
     "pd.DataFrame({\n",
     "    \"framework\": [\"hvPlot\", \"HoloViews\", \"Panel\"], \n",
     "    \"stars\": [700, 2400, 2600]\n",
-    "}).hvplot.bar(x=\"framework\", y=\"stars\", color=\"gold\", title=\"Bar Plot of Github Stars as of March 2023\", ylabel=\"⭐\")"
+    "}).hvplot.bar(x=\"framework\", y=\"stars\", color=\"gold\", title=\"Bar Plot of Github Stars\", ylabel=\"⭐\")"
```

Comment:
Removed the date, just to avoid having to update it in the future.

Reply:
Its ok with me. I actually put it in though such that people Boes this is older. Now i would expect the data to be up to date 😂

Reply:
Yep it's difficult to have both ^^